apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -474,6 +508,60 @@ void _wt_connect(void *arg)
 	WT_LEAVE;
 }
 
+void _wt_scan_connect(void *arg)
+{
+	WT_ENTER;
+	WT_LOG(TAG, "--> scan_connect\n");
+	g_scan_join = 1;
+	struct wt_options *ap_info = (struct wt_options *)arg;
+	wifi_manager_scan_config_s sconfig;
+	sconfig.channel = 0;
+	sconfig.ssid_length = strlen(ap_info->ssid);
+	strncpy(sconfig.ssid, ap_info->ssid, sconfig.ssid_length + 1);
```

Comment:
`ap_info` is got from parameter `arg`. So please check whether `arg` is valid or not to avoid wrong memory access.

Reply:
it's guaranteed that arg is valid.